src/assets/Azure.Core.Shared/UriBuilderExtensions.cs

```diff
@@ -87,12 +87,12 @@ public static void AppendQuery(this RequestUriBuilder builder, string name, byte
             builder.AppendQuery(name, Convert.ToBase64String(value), escape);
         }
 
-        public static void AppendQuery(this RequestUriBuilder builder, string name, IEnumerable<string> value, bool escape = true)
+        public static void AppendQueryDelimited<T>(this RequestUriBuilder builder, string name, IEnumerable<T> value, string delimiter, bool escape = true)
```

Comment:
Will this not work as an overload for AppendQuery?

Reply:
I don't want it to accidentally match AppendQuery with string value (because it implements IEnumerable<char>).